Makefile

```diff
@@ -238,28 +245,20 @@ integration: build-race
 
 testall: fulltest integration
 
-# generated files we should make sure we clean
-GENERATED_FILES := daemon/algod/api/bundledSpecInject.go \
-	daemon/algod/api/lib/bundledSpecInject.go \
-	daemon/kmd/lib/kmdapi/bundledSpecInject.go \
-	$(ALGOD_API_SWAGGER_SPEC) $(ALGOD_API_SWAGGER_SPEC).validated \
-	$(KMD_API_SWAGGER_SPEC) $(KMD_API_SWAGGER_SPEC).validated
-
 clean:
 	go clean -i ./...
 	rm -f $(GOPATH1)/bin/node_exporter
-	rm -f $(GENERATED_FILES)
```

Comment:
I don't think so. That would cause more issues, since cleaning them without regenerating them would cause build failures ( keep in mind that I've removed the dependencies on these, since they're alway there ).
That's the reason I added `rebuild_swagger` target.